### PR TITLE
dev(sg): add run redis-postgres via compose

### PR DESCRIPTION
**Pull request by bobheadxi**
because I use this :) only works with override config for the `PG*` variables. not sure how to just make this work out-of-the-box

```
❯ sg run redis-postgres
```

### CODE REVIEW

File: sg.config.yaml [hunk 1]
```diff
@@ -483,6 +483,17 @@ commands:
...
 
+  redis-postgres:
+    # Add the following overwrites to your sg.config.overwrite.yaml to use the docker-compose
+    # database:
+    #
+    #   env:
+    #       POSTGRES_HOST: localhost
+    #       PGPASSWORD: sourcegraph
+    #       PGUSER: sourcegraph
+    #
```
Comment (mrnugget) [inline sg.config.yaml:494]: You could also add an overwrite to add `redis-postgres` to the `default` command set.

**Review by mrnugget — APPROVED** [review]
Looks good to me! But @fkling added the `compose.yaml` file so I want him to take a look before his setup breaks on Monday morning :)

**Comment by fkling** [comment]
> only works with override config for the PG* variables. 

Problably depends on what the default values are set to, if at all. They are set in `dev/start.sh`: https://sourcegraph.com/github.com/sourcegraph/sourcegraph@1d17d01c7c9d74a1fee198913104490ee5ef7310/-/blob/dev/start.sh?L36-41

Otherwise I have no problem with renaming the file 😆

**Review by fkling — APPROVED** [review]

**Comment by bobheadxi** [comment]
> Problably depends on what the default values are set to, if at all. They are set in dev/start.sh: https://sourcegraph.com/github.com/sourcegraph/sourcegraph@1d17d01c7c9d74a1fee198913104490ee5ef7310/-/blob/dev/start.sh?L36-41

It seems to be different with `sg start`, but I'll leave that question for another time 😛

**Pull request merged**